share/errmsg-utf8.txt

```diff
@@ -19271,6 +19271,9 @@ ER_PARALLEL_DOUBLEWRITE_WRITE_ERROR
 ER_XB_MSG_6
   eng "Key rotation of encrypted session temporary tablespace %s failed"
 
+ER_XB_MSG_7
```

Comment:
New error messages can try to use meaningful mnemonics (the ER_IB_MSG_1234 existed because it was a mass change as far as I can tell)

Reply:
I see, let me add better name. I thought, it has to be sequence :)